pina/problem/zoo/supervised_problem.py

```diff
@@ -1,9 +1,22 @@
+import torch
 from pina.problem import AbstractProblem
 from pina import Condition
 from pina import Graph
+from pina import LabelTensor
 
 
-class SupervisedProblem(AbstractProblem):
+class SupervisedProblem:
+
+    def __new__(cls, *args, **kwargs):
+
+        if sorted(list(kwargs.keys())) == sorted(["input_", "output_"]):
```

Comment:
I would start writing `input`, `target`. And for graph make another problem. supervised problems are defined on an input and on a target, having only one value is misleading